jeecg-boot-module-system/src/main/resources/application-dev.yml

```diff
@@ -98,7 +98,7 @@ spring:
         master:
           url: jdbc:mysql://127.0.0.1:3306/elnido?characterEncoding=UTF-8&useUnicode=true&useSSL=false
           username: root
-          password: jackingyu123
```

Comment:
no need to submit this...you can keep this on your local,dev is for local devlopement